test/jdk/java/awt/MenuItem/GiantFontTest.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright (c) 2007, 2024, Oracle and/or its affiliates. All rights reserved.
```

Comment:
```suggestion
 * Copyright (c) 2002, 2024, Oracle and/or its affiliates. All rights reserved.
```
This test was writted in 2002 for [JDK-4700350](https://bugs.openjdk.org/browse/JDK-4700350).

Could you also add `@bug 4700350` to jtreg tags?

https://github.com/openjdk/jdk/blob/ae39a6603c6c33a36dce30c3290a634b08a6bf05/src/java.desktop/windows/native/libawt/windows/awt_MenuItem.cpp#L513-L516

Reply:
Thanks for finding the bugId. I was unable to find the details.